config/breadcrumbs.rb

```diff
@@ -94,12 +94,22 @@
   parent :editorial_root
 end
 
-crumb :public_news_articles do |section|
-  link 'News', section_news_articles_path(section.slug)
-  parent :public_node, section.home_node
+crumb :public_news_articles do
+  link 'News', news_articles_path
+  parent :public_node, Node.root_node
 end
 
 crumb :public_news_article do |node|
   link node.name, public_node_path(node)
-  parent :public_news_articles, node.section
+  parent :public_news_articles
+end
+
+crumb :section_news_articles do |section|
+  link 'News', nil
```

Comment:
On the off chance that eventually there are children of this crumb, we should specify the path for the link here. Similarly for public_news_aggregator.

Also, what's the difference between public_news_articles and public_news_aggregator? Are they actually the same?